crates/rust-analyzer/src/global_state.rs

```diff
@@ -166,6 +166,14 @@ pub(crate) struct GlobalStateSnapshot {
     pub(crate) flycheck: Arc<[FlycheckHandle]>,
 }
 
+#[derive(Debug, Clone)]
+pub(crate) enum FlycheckStatus {
+    Unknown,
```

Comment:
`Unknown` seems unnecessary, using `Finished` in the places where `Unknown` is used works just fine I'd say

Reply:
Ok